microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -894,6 +905,48 @@ def verify_boot_error_fail_warnings(self, node: Node) -> None:
             f" {node.os.name} {node.os.information.version}"
         ).is_empty()
 
+    @TestCaseMetadata(
+        description="""
+        This test will check ERROR, WARNING messages from /var/log/cloud-init.log
+        and also check cloud-init exit status.
+
+        Steps:
+        1. Get ERROR, WARNING messages from /var/log/cloud-init.log.
+        2. If any unexpected ERROR, WARNING messages or non-zero cloud-init status
+         fail the case.
+        """,
+        priority=1,
```

Comment:
Move it to P2, because it's distro specific.

Reply:
change as suggested